packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/network/dedicatedCloud-datacenter-network.routing.js

```diff
@@ -0,0 +1,31 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'app.dedicatedCloud.details.datacenter.details.network',
+    {
+      url: '/network',
+      views: {
+        pccDatacenterView: {
+          component: 'ovhManagerDedicatedCloudDatacenterNetwork',
+        },
+      },
+      redirectTo: (transition) =>
+        transition
+          .injector()
+          .getAsync('resources')
+          .then((resources) =>
+            resources.length === 0
+              ? {
+                  state:
+                    'app.dedicatedCloud.details.datacenter.details.network.onboarding',
+                }
+              : false,
+          ),
+      resolve: {
+        resources: /* @ngInject */ ($http) =>
+          $http.get('/dedicatedCloud').then(({ data }) => data),
```

Comment:
```suggestion
          $http.get('/dedicatedCloud').then(({ data }) => data),
```

Request should be in a service